### ISSUE TITLE
`MetricPublisher` is using a deprecated API

### ISSUE THREAD

**Issue by maiksensi** [codecentric/chaos-monkey-spring-boot#128]
When packaging CMSB we get a deprecation warning. We should investigate what API is deprecated and see whether we can port to a newer API. 

```
[WARNING] *chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/component/MetricEventPublisher.java: /Users/maiksensi/git/forkchaos/chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/component/MetricEventPublisher.java uses or overrides a deprecated API.
[WARNING] *chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/component/MetricEventPublisher.java: Recompile with -Xlint:deprecation for details.
```

**Comment by Abijithkrishna** [comment]
Hi, can I take this up

**Comment by WtfJoke** [comment]
Just linking this to #131

**Comment by WtfJoke** [comment]
Fixed by #137
